builtin/providers/aws/resource_aws_cloudwatch_metric_alarm_test.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import(
+    "fmt"
+//    "math/rand"
+//    "strings"
+    "testing"
+//    "time"
+//
+    "github.com/awslabs/aws-sdk-go/aws"
+    "github.com/awslabs/aws-sdk-go/service/cloudwatch"
+    "github.com/hashicorp/terraform/helper/resource"
+    "github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSCloudWatchMetricAlarm_basic(t* testing.T) {
+    var alarm cloudwatch.MetricAlarm
+
+    resource.Test(t, resource.TestCase{
+        PreCheck:       func () { testAccPreCheck(t) },
+        Providers:      testAccProviders,
+        CheckDestroy:   testAccCheckAWSCloudWatchMetricAlarmDestroy,
+        Steps:          []resource.TestStep{
+            resource.TestStep{
+                Config: testAccAWSCloudWatchMetricAlarmConfig,
+                Check:  resource.ComposeTestCheckFunc(
+                    testAccCheckCloudWatchMetricAlarmExists("aws_cloudwatch_metric_alarm.foobar", &alarm),
```

Comment:
Same note as above. I'm sure this is just a copy-paste from another test that's doing this, and should be cleaned up, but going forward let's not introduce more :smile: 

Unless you intend on checking it's properties, it can be removed.